### ISSUE TITLE
{Freeway-Bug}[Adaptive cards-Sample tab -Inputs with validation]: At reflow settings, the checkbox label is not fully visible on the Inputs with Validation page.

### ISSUE THREAD

**Issue by HasnuddinMohammed** [microsoft/AdaptiveCards#9159]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test environment:**
OS: Win11 
OS Version: 23H2 (OS Build 22631.5189)  
URL: https://adaptivecards.io/  
Edge dev Browser version: 137.0.3255.0 (Official build) dev (64-bit)  
Screen reader: Narrator. 

**Pre requisites:**  
100% DPI Scaling, set the browser to 200% and bring viewport to 320px 

**Repro Steps:**
1.Open URL in the Edge dev browser & sign in with valid credentials  
2.Activate the samples tab 
3. Activate the Inputs with validation and observe the issue. 

**Actual Result:**  
At reflow settings, label of checkbox (long text) like I accept the terms … is not fully visible in the Inputs with validation page. 

 **Expected Result:**  
At reflow settings, label of checkbox (long text) like I accept the terms should fully visible in the Inputs with validation page. 

**User Experience:** 
Low vision users depends on the zoom levels will be impacted and may find it difficult to keep track of their place while reading, if the content of the check boxes are not visible at 400% zoom. 

**MAS Reference:**  
MAS 1.4.10 – Reflow 

### Screenshots

[Link](https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-perumallab_microsoft_com/Efaa4LPwUqxFtrRss1pe9ccBwyRW8vgycobryMb_nq8wVg?e=BEjduU&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D)

### Card JSON

```JSON
NA
```
